hotham-simulator/src/state.rs

```diff
@@ -216,7 +204,7 @@ impl State {
 
         let dt = delta.as_secs_f32();
         let movement_speed = 2f32 * dt;
-        let mouse_sensitivity = 4f32 * dt;
+        let mouse_sensitivity = 40f32 * dt;
```

Comment:
not sure why but I had to massive bump the mouse_sensitivity up

Reply:
`dt` should not be here because we don't want our framerate to decide how fast the mouse turns the view.

Reply:
yeah good point, we probably don't really need it for camera directional movement either